http-server/src/main/java/org/triplea/http/ServerApplication.java

```diff
@@ -165,7 +160,7 @@ public void run(final AppConfig configuration, final Environment environment) {
         .getUserProperties()
         .put(PlayerConnectionWebSocket.GAME_LISTING_QUEUE_KEY, gameListingEventQueue);
 
-    final var messagingService = MessagingServiceFactory.build(jdbi, chatSessions, chatters);
+    final var messagingService = ChatMessagingService.build(jdbi, chatSessions, chatters);
```

Comment:
Its not clear to me if the variable type changed here? Should the var name be updated to `chatMessagingService`?

Reply:
It's not a super critical update as `ChatMessagingService` is still a 'messagingService'. `MessagingServiceFactory` was renamed as it was too generic, it really was 'chat' messages, and *not* things like network communication, internal events, queues, etc..

Let's keep an eye on this for the branch in #6096, updating this here would conflict with that, that is my only hesitation to not update it now.

The main thrust of this PR is that we are going from factory classes, to a factory method.